standard/links_stream.lua

```diff
@@ -161,27 +256,24 @@ end
 ---@return string?, integer?
 function StreamKey:_fromLegacy(input)
 	for _, platform in pairs(StreamLinks.countdownPlatformNames) do
-		-- The intersection of values in countdownPlatformNames and keys in streamPlatformLookupNames
-		-- are not valid platforms. E.g. "twitch2" is not a valid platform.
-		if not StreamLinks.streamPlatformLookupNames[platform] then
-			-- Check if this platform matches the input
-			if string.find(input, platform .. '%d-$') then
-				local index = 1
-				-- If the input is longer than the platform, there's an index at the end
-				-- Eg. In "twitch2", the 2 would the index.
-				if #input > #platform then
-					index = tonumber(input:sub(#platform + 1)) --[[@as integer]]
-				end
-				return platform, index
+		if string.find(input, platform .. '%d-$') then
+			local index = 1
+			-- If the input is longer than the platform, there's an index at the end
+			-- Eg. In "twitch2", the 2 would be the index.
+			if #input > #platform then
+				index = tonumber(input:sub(#platform + 1)) or index
```

Comment:
Should probably error here instead of `or index`? 
Because it would mean the suffix is non-numeric. 
While that shouldn't even be possible with the `%d` matching?